### PR TITLE
Update apps.json

### PR DESCRIPTION
**Pull request by siparker**
fixed google tag manager detection

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -2636,8 +2636,8 @@
 			"cats": [
 				42
 			],
-			"env": "^googletag$",
-			"html": "googletagmanager\\.com/ns\\.html[^>]+></iframe>",
```
Comment (AliasIO) [inline src/apps.json]: Did you mean to remove these?

Comment (siparker) [inline src/apps.json]: yes the env seemed to make no difference in detection so saved the line to keep it as small as possible. 
the html line was not working at all and never detected anything.

Comment (AliasIO) [inline src/apps.json]: If the `env` works than we should keep it in, it will identify the app when it's loaded through a different script.

Comment (siparker) [inline src/apps.json]: i did test with just the env in and it seems to do nothing?
in fairness i dont know how to check env variables properly though so if you think it should stay in by all means please just edit the html to be the script one instead.

Comment (q--) [inline src/apps.json]: @siparker You can see if an environment variable is present simply by typing its name into your  browser console. If it's autocompleted, then the variable is present. If it isn't, then the variable is not present but you'll still want to make sure that the variable is present *nowhere*, even in older versions and/or on different pages. Also, Wappalyzer patterns are case-insensitive but variable names _are_!

And as a rule of a thumb: don't remove detection patterns unless you're absolutely, positively sure it does not manage to detect a single version of the targeted application (even if that version was deprecated several years ago) or if you've written a replacement rule that detects the _same_ property better.

Comment (siparker) [inline src/apps.json]: hi. thats great info. would def be worth adding to the wiki!!

in that case the env variable is indeed present on the pages i have to test. should i try to modify the pull request or did you just want to get that one line?

Also any other recommendations on how to identify possible env variables if you dont know what they are called?

**Pull request closed without merging**